### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#314] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322694-236486c8-c836-4646-b45a-723f61a93aef.png)

**Comment by Exyss** [comment]
L'errore è nell'applicazione del passo induttivo:

Ponendo n = 2, otteniamo i due insiemi A1 = {a1} e A2 = {a2}, i quali non possiedono alcun elemento in comune, dunque non è possibile instaurare il "ponte di equivalenza" previsto dal passo induttivo stesso.

La tesi dunque è invalida.
